src/controllers/scenario/index.ts

```diff
@@ -1,11 +1,4 @@
-import {
-  Controller,
-  Get,
-  Query,
-  Path,
-  Route,
-  Response,
-} from 'tsoa'
+import { Controller, Get, Query, Path, Route, Response } from 'tsoa'
```

Comment:
hang on I'm confused. Why wasn't this failing linting before?

Reply:
`"lint": "eslint ./src"` in package scripts  is the wrong command. I think it should be `eslint .` and then we add certain directories like `build` to `.eslintignore`. Will do in a separate PR, there's loads of linting errors

Reply:
https://digicatapult.atlassian.net/browse/DSBD-72